woocommerce/woocommerce
# Image Optimization: Compress pattern placeholder images

### Changes proposed in this Pull Request:

This PR is part 1 of 2. Here I have compressed all of the existing images for pattern placeholders which has **reduced the directory down from 10MB to 4.1MB**. In a separate PR I will look at image dimensions to further optimize which will take a little more time for regression testing.

Related to https://github.com/woocommerce/woocommerce/issues/50379



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check comparison of image quality in this PR to ensure no significant degradation.
2. Spot check pattern placeholders to ensure images still render as expected.
